src/lib/transport.js

```diff
@@ -27,6 +28,8 @@ function createPacketAndCondition ({
   destinationAccount,
   secret,
   data,
+  headers,
+  unsafeHeaders,
```

Comment:
The parameters of createPacketAndCondition in IPR and PSK are passed directly through to the Transport class. I'll add a test to make sure that additional headers can be specified though.